CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Release Date: UNRELEASED Valhalla 3.0.8
```

Comment:
need to update the header with the new version

Reply:
I thought we do that when re release it

Reply:
NM we'll make a release later!